CHANGES.rst

```diff
@@ -8,6 +8,8 @@
 - Fix UnboundLocalError when running fsoids.py script.
   See `issue 268 <https://github.com/zopefoundation/ZODB/issues/285>`_.
 
+- Rework `fsrefs` script to work significantly faster by optimizing how it does
+  IO. See `PR 340 <https://github.com/zopefoundation/ZODB/pull/340>`.
```

Comment:
```suggestion
- Rework ``fsrefs`` script to work significantly faster by optimizing how it does
  IO. See `PR 340 <https://github.com/zopefoundation/ZODB/pull/340>`_.
```

Double-backtictks are literal code; single backticks are the default object reference. 

Links need a trailing underscore to be valid.

Reply:
Thanks for catching this and sorry I did not verify that rst -> html conversion works correctly after my edits. I've integrated your suggestions into this pull-requst.

By the way, now, after verifying rst, I see this:

```
(neo) (z4-dev) (g.env) kirr@deco:~/src/wendelin/z/ZODB$ rst2html --link-stylesheet CHANGES.rst x.html
CHANGES.rst:3: (WARNING/2) Duplicate explicit target name: "issue 268".
``` 

The warning it emits is about the following text where link for `issue 286` seems to be incorrect:

```
- Fix UnboundLocalError when running fsoids.py script.
  See `issue 268 <https://github.com/zopefoundation/ZODB/issues/285>`_.
```

It should be changed to

```diff
--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -6,7 +6,7 @@
 ==================
 
 - Fix UnboundLocalError when running fsoids.py script.
-  See `issue 268 <https://github.com/zopefoundation/ZODB/issues/285>`_.
+  See `issue 285 <https://github.com/zopefoundation/ZODB/issues/285>`_.
 
 - Rework ``fsrefs`` script to work significantly faster by optimizing how it does
   IO. See `PR 340 <https://github.com/zopefoundation/ZODB/pull/340>`_.
```

If it is ok I can do and push this fix to master after merging this PR.

Reply:
Sure, though it'd be fine to just include the fix here.

FWIW, I usually use @mgedmin 's [restview](https://pypi.org/project/restview/) to verify rendering because it has an option that makes it do what PyPI does, and because it checks the contents of the markup as they will be seen by PyPI — many zopefoundation projects, including ZODB, create this output by concatenating README.rst and CHANGES.rst.

```console
$ restview --long --pypi 
Listening on http://localhost:62676/
127.0.0.1 - - [29/Mar/2021 07:26:22] "GET / HTTP/1.1" 200 -
```

Reply:
Thanks for restview link. To do the verification I used to do something like

```
$ python setup.py --long-description |rst2html - x.html
```

but restview indeed might be more convenient.

Let's have changelog fixup coming as separate patch (out of this PR) not to mix things.

Reply:
I've fixed "issue 268" in https://github.com/zopefoundation/ZODB/commit/2798502e. Hope it is ok.